tools/clang/lib/Headers/hlsl/linalg.h

```diff
@@ -0,0 +1,172 @@
+// Header for linear algebra APIs.
```

Comment:
Should this be in `tools/clang/lib/Headers/hlsl/dx` since it is dx specific?

Reply:
Moved.